backend/app/templates/app/index.html

```
@@ -634,13 +634,18 @@ <h2>
 {% endblock content %}
 
 <div id="portals-container" hx-swap-oob="true">
+<calcite-list>
   {% for item in portal %}
-    <calcite-action text="{{ item.0 }}" {% if item.1 == 'portal' %}icon="portal"
-                    {% else %}icon="arcgis-online"{% endif %} text-enabled
-                    {% if instance == item.0 %}active {% endif %}
-                    hx-get="{% url 'enterpriseviz:viz' instance=item.0 %}" hx-target="#mainbodycontent"
+    <calcite-list-item id="portal-{{ item.0|slugify }}" text="{{ item.0 }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)